README.md

```diff
@@ -1,137 +1,123 @@
 This repository details how changes to Solid may be proposed and accepted.
 
 # Solid Specification
-The following is a description about how changes to the specifications in the Solid ecosystem may be proposed and accepted.
+The following describes how changes to the specifications in the Solid ecosystem may be proposed and accepted. Anyone may submit a proposal to alter this process; such proposals will be reviewed only by [Tim Berners-Lee](https://github.com/timbl), who is the Solid Director.
 
-The current development on the Solid ecosystem is carried out in the [Solid Specifications](https://github.com/solid/specification) repository. The [Unofficial Draft](https://github.com/solid/solid-spec) predating the current development is still available for historical reference.
+Development in the Solid ecosystem occurs in the [Solid Specifications](https://github.com/solid/specification) repository. The [Unofficial Draft](https://github.com/solid/solid-spec) predating current development is still available for historical reference.
 
-Anyone may participate in [this process](https://github.com/solid/culture). Please read the [Code of Conduct](code-of-conduct.md) before doing so.
+Anyone may participate in this process. Please read the [Code of Conduct](code-of-conduct.md) before doing so.
+
+## Editorial Team
+
+The Solid [Editorial Team](https://github.com/orgs/solid/teams/editors) is responsible for the implementation of the Solid Specification process. Members of the Editorial Team are appointed by the Solid Director. The Editorial Team is comprised of all the Editors appointed by the Solid Director, who are listed at [editors.md](editors.md), along with their assignments, contact details, and affiliations. The Editorial Team shepherds Solid and coordinates with those who participate in Solid Panels and with Stakeholders and Implementers. The Solid Director may also appoint support personnel on an as-needed basis, such as a Release Manager, for organizing the workflow of the specification process.
```

Comment:
Linked. And the comment about the editorial team is intentionally vague so that they don't overstep their bounds but also so it's not too prescriptive.